Makefile

```diff
@@ -118,26 +118,26 @@ go-gpt2:
 	git clone --recurse-submodules https://github.com/go-skynet/go-gpt2.cpp go-gpt2
 	cd go-gpt2 && git checkout -b build $(GOGPT2_VERSION) && git submodule update --init --recursive --depth 1
 	# This is hackish, but needed as both go-llama and go-gpt4allj have their own version of ggml..
-	@find ./go-gpt2 -type f -name "*.c" -exec sed -i'' -e 's/ggml_/ggml_gpt2_/g' {} +
-	@find ./go-gpt2 -type f -name "*.cpp" -exec sed -i'' -e 's/ggml_/ggml_gpt2_/g' {} +
-	@find ./go-gpt2 -type f -name "*.h" -exec sed -i'' -e 's/ggml_/ggml_gpt2_/g' {} +
-	@find ./go-gpt2 -type f -name "*.cpp" -exec sed -i'' -e 's/gpt_print_usage/gpt2_print_usage/g' {} +
-	@find ./go-gpt2 -type f -name "*.h" -exec sed -i'' -e 's/gpt_print_usage/gpt2_print_usage/g' {} +
-	@find ./go-gpt2 -type f -name "*.cpp" -exec sed -i'' -e 's/gpt_params_parse/gpt2_params_parse/g' {} +
-	@find ./go-gpt2 -type f -name "*.h" -exec sed -i'' -e 's/gpt_params_parse/gpt2_params_parse/g' {} +
-	@find ./go-gpt2 -type f -name "*.cpp" -exec sed -i'' -e 's/gpt_random_prompt/gpt2_random_prompt/g' {} +
-	@find ./go-gpt2 -type f -name "*.h" -exec sed -i'' -e 's/gpt_random_prompt/gpt2_random_prompt/g' {} +
-	@find ./go-gpt2 -type f -name "*.cpp" -exec sed -i'' -e 's/json_/json_gpt2_/g' {} +
+	@find ./go-gpt2 -type f -name "*.c" -exec sh -c "sed -e 's/ggml_/ggml_gpt2_/g' {} > {}.tmp && mv {}.tmp {}" \; -print | wc -l
+	@find ./go-gpt2 -type f -name "*.cpp" -exec sh -c "sed -e 's/ggml_/ggml_gpt2_/g' {} > {}.tmp && mv {}.tmp {}" \; -print | wc -l
+	@find ./go-gpt2 -type f -name "*.h" -exec sh -c "sed -e 's/ggml_/ggml_gpt2_/g' {} > {}.tmp && mv {}.tmp {}" \; -print | wc -l
+	@find ./go-gpt2 -type f -name "*.cpp" -exec sh -c "sed -e 's/gpt_print_usage/gpt2_print_usage/g' {} > {}.tmp && mv {}.tmp {}" \; -print | wc -l
+	@find ./go-gpt2 -type f -name "*.h" -exec sh -c "sed -e 's/gpt_print_usage/gpt2_print_usage/g' {} > {}.tmp && mv {}.tmp {}" \; -print | wc -l
+	@find ./go-gpt2 -type f -name "*.cpp" -exec sh -c "sed -e 's/gpt_params_parse/gpt2_params_parse/g' {} > {}.tmp && mv {}.tmp {}" \; -print | wc -l
+	@find ./go-gpt2 -type f -name "*.h" -exec sh -c "sed -e 's/gpt_params_parse/gpt2_params_parse/g' {} > {}.tmp && mv {}.tmp {}" \; -print | wc -l
+	@find ./go-gpt2 -type f -name "*.cpp" -exec sh -c "sed -e 's/gpt_random_prompt/gpt2_random_prompt/g' {} > {}.tmp && mv {}.tmp {}" \; -print | wc -l
+	@find ./go-gpt2 -type f -name "*.h" -exec sh -c "sed -e 's/gpt_random_prompt/gpt2_random_prompt/g' {} > {}.tmp && mv {}.tmp {}" \; -print | wc -l
+	@find ./go-gpt2 -type f -name "*.cpp" -exec sh -c "sed -e 's/json_/json_gpt2_/g' {} > {}.tmp && mv {}.tmp {}" \; -print | wc -l
```

Comment:
why piping to `wc -l`?

Reply:
I want to print the number of files that have been replaced by the command. 
I add it because I do not know if it works well or not. And if the number goes to change I know maybe something wrong here. Is that useful?

Here is the example:
```
git clone --recurse-submodules https://github.com/go-skynet/bloomz.cpp bloomz
Cloning into 'bloomz'...
remote: Enumerating objects: 287, done.
remote: Counting objects: 100% (83/83), done.
remote: Compressing objects: 100% (53/53), done.
remote: Total 287 (delta 35), reused 64 (delta 28), pack-reused 204
Receiving objects: 100% (287/287), 2.12 MiB | 2.11 MiB/s, done.
Resolving deltas: 100% (136/136), done.
2
4
8
4
8
```